admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -642,6 +645,25 @@ var specificPrices = (function() {
           $('#form_step2_specific_price_sp_reduction_tax').show();
         }
       });
+
+      /* enable price field only when needed */
+      leaveInitialPrice.on('click', function togglePriceField() {
+        productPriceField.prop('disabled', $(this).is(':checked'))
+          .val('')
+        ;
+      });
+
+      /* enable tax type field only when reduction by amount is selected */
+      discountTypeField.on('change', function toggleDiscountTaxField() {
+        var uglySelect2Selector = $('#select2-form_step2_specific_price_sp_reduction_tax-container').parent().parent();
```

Comment:
Is there no other way to get the element you want ? I'm pretty sure this will be broken shortly.